docs/source/gpus/multigpu-partition-access.md

```diff
@@ -0,0 +1,94 @@
+# Access to Multi-GPU Partition
+The `multigpu` partition in the HPC cluster allows users for extensive parallel processing. This setup is ideal for applications that require significant computational power, such as deep learning, scientific simulations, and large-scale data analysis.
+
+## Applying for Multi GPU reservation
+
+### Step 1
+
+- Use the partition application form link in here: [HPC Partition Request](https://bit.ly/NURC-PartitionAccess)
+- Select under "Partition Type" - `Multigpu - Partition (Testing Access)`
+- Fill in the number of GPUs and the type of GPU you'd like to test on. Please also provide a short description of your expected testing workload.
+- Submit the form.
+- RC will then contact you with further details for accessing a multi-gpu node. RC team will need an estimate of the duration needed for testing (preferably less than 24 hrs.)
```

Comment:
RC -> The Research Computing team

I would just expand this and not use the abbreviation because not sure if everyone will know what RC is if they just jump to this page.